packages/twenty-front/src/modules/settings/data-model/fields/forms/select/components/SettingsDataModelFieldSelectFormOptionRow.tsx

```diff
@@ -126,7 +156,10 @@ export const SettingsDataModelFieldSelectFormOptionRow = ({
           onChange({
             ...option,
             label,
-            value: getOptionValueFromLabel(label),
+            value:
+              option.value === getOptionValueFromLabel(option.label)
```

Comment:
Can you explain your condition rationale?
Why not use isAdvancedModeEnabled here to decide whether or not label update should trigger value update?

Reply:
Hello @ijreilly 
Before the changes, if the user edited the label, the API value changed accordingly, right? now we are maintaining this synchronization if the user does not change the API value, so the user do not need to write everything again for each API value, if the user just wants to edit few values

Reply:
Got it thanks, I will add an intermediary const to make that clear